Containers/nextcloud/config/apps.config.php

```diff
@@ -12,4 +12,5 @@
               'writable' => true,
       ),
   ),
+  'appsallowlist' => getenv('APPS_ALLOWLIST') ? explode(" ", getenv('APPS_ALLOWLIST')) : [],
```

Comment:
I am not completely sure about this syntax. Does this look good?

Reply:
👍

Reply:
Thanks! Does the rest of the PR also look good? :) (no testing required)